README.md

```diff
@@ -773,6 +810,89 @@ The `WHERE`describes in which part of the app the integer is used. If
 it is used in more than one screen use `all`. The `DESCRIPTION` should
 tell the reader what the actual purpose of the integer is.
 
+<a name="naming--colors"></a><a name="11.6"></a>
+- [11.6](#naming--colors) Colors should be named after the following rules: Do not use prefixes such as dark, light, bright. You cannot ensure that you never need e.g. an even darker blue than your current darkblue in future. Also, do not define a color multiple times because it makes it difficult to apply changes. Instead create a distinct color palette and reference from there. Make sure that you use one of the formats below.
+
+Basic colors like white, black, grey are necessary for fonts, lines, borders, shadows etc. and should be named after the following pattern:
+
+```
+<HUE>_<LIGHTNESS>
```

Comment:
What does lightness mean? Is this the opacity? We also should propose the following

<COLORNAME>(_alpha_<OPACITY>)

so that we can also use this plugin (https://plugins.jetbrains.com/plugin/10422-name-that-color)